ratatui-widgets/src/mascot.rs

```diff
@@ -86,7 +86,7 @@ impl Default for RatatuiMascot {
             rat_eye_color: Color::Indexed(236),     // dark_charcoal #303030
             rat_eye_blink: Color::Indexed(196),     // red #ff0000
             term_color: Color::Indexed(232),        // vampire_black #080808
-            term_border_color: Color::Indexed(244), // gray  #808080
+            term_border_color: Color::Indexed(237), // gray  #808080
```

Comment:
@joshka Note that the comment should, in theory, be 

```rs
term_border_color: Color::Indexed(237), // #3a3a3a black_olive
```

Reply:
Ha - oops D: